wallet/src/account/mod.rs

```diff
@@ -128,6 +147,93 @@ impl Account {
         Ok(tx)
     }
 
+    fn get_vrf_key(&self) -> WalletResult<(VRFPrivateKey, VRFPublicKey)> {
+        let vrf_key_path = make_path_to_vrf_key(&self.chain_config, self.account_index());
+        let private_key = self.key_chain.get_private_key_for_path(&vrf_key_path)?.private_key();
+        let vrf_keys =
+            VRFPrivateKey::new_from_bytes(private_key.as_bytes(), VRFKeyKind::Schnorrkel)
+                .expect("should not fail because private keys are always 32 bytes");
```

Comment:
This transformation is incorrect and can lead to issues. Not all points on a curve are valid in another. We should allow storing VRF keys properly.